FirebaseAdmin/FirebaseAdmin/Messaging/InstanceIdClient.cs

```diff
@@ -151,5 +187,23 @@ private class InstanceIdServiceRequest
             [JsonProperty("registration_tokens")]
             public List<string> RegistrationTokens { get; set; }
         }
+
+        private class InstanceIdServiceResponse
```

Comment:
Ah... I see what you mean. You're also using this in `TopicManagementResponse`. In that case lets move this to its own file (like you had it earlier), but lets make it `internal`.